.changeset/large-fireants-help.md

```diff
@@ -0,0 +1,5 @@
+---
+'@mobile-reality/react-native-select-pro': minor
+---
+
+add second type of options data: Section List
```

Comment:
you can add a code example here with descriptive text e.g. Now you can grouping your options